src/js/components/Box/Box.js

```diff
@@ -230,15 +233,14 @@ const Box = forwardRef(
             !border.find((b) => b.side === 'between')) &&
           gap
         }
-        kindProp={kind}
-        overflowProp={overflow}
-        wrapProp={wrap}
-        widthProp={width}
-        heightProp={height}
-        responsive={responsive}
-        tabIndex={adjustedTabIndex}
+        $kind={kind}
+        $overflow={overflow}
+        $wrap={wrap}
+        $width={width}
+        $height={height}
+        $responsive={responsive}
```

Comment:
small detail but can we alphabetize these?

Reply:
Yes, good note